vuu/src/main/scala/org/finos/vuu/core/module/simul/SimulationModule.scala

```diff
@@ -77,13 +77,28 @@ class InstrumentsService(val table: DataTable, val providerContainer: ProviderCo
     NoAction()
   }
 
-  override def menuItems(): ViewPortMenu = ViewPortMenu("Test Menu",
+  override def menuItems(): ViewPortMenu = ViewPortMenu(
+    ViewPortMenu("Block Actions (selected rows)",
```

Comment:
Why so many quotes on the filter definition..?

Reply:
fixed